frontend/src/scenes/insights/Funnels.stories.tsx

```diff
@@ -67,33 +67,35 @@ FunnelLeftToRightBreakdownEdit.parameters = {
     testOptions: { waitForSelector: ['[data-attr=funnel-bar-vertical] .StepBar', '.PayGateMini'] },
 }
 
-export const FunnelTopToBottom: Story = createInsightStory(
-    require('../../mocks/fixtures/api/projects/team_id/insights/funnelTopToBottom.json')
-)
-FunnelTopToBottom.parameters = {
-    testOptions: { waitForSelector: ['[data-attr=funnel-bar-horizontal] .funnel-bar-percentage', '.PayGateMini'] },
-}
-export const FunnelTopToBottomEdit: Story = createInsightStory(
-    require('../../mocks/fixtures/api/projects/team_id/insights/funnelTopToBottom.json'),
-    'edit'
-)
-FunnelTopToBottomEdit.parameters = {
-    testOptions: { waitForSelector: ['[data-attr=funnel-bar-horizontal] .funnel-bar-percentage', '.PayGateMini'] },
-}
+// TODO: this test is very flakey due to relying on ResizeObserver. Either fix it, or remove it completely.
```

Comment:
Let's just remove. There's no way anyone will straight up uncomment this ever.
I mean, technically, there's a way, but we know it won't happen and those stories will probably be straight-up broken as-is then.